k8s-ruby.gemspec

```
@@ -29,7 +29,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "dry-configurable"
   spec.add_runtime_dependency "recursive-open-struct", "~> 1.1.3"
   spec.add_runtime_dependency "hashdiff", "~> 1.0.0"
-  spec.add_runtime_dependency "jsonpath", "~> 0.9.5"
+  spec.add_runtime_dependency "jsonpath", "~> 1.1.0"
```

What if we remove the pinning?

---

I don't mind removing it, makes no difference to me.

---

I suggest keeping this to ensure that at least it stays locked to the major version:
```suggestion
  spec.add_runtime_dependency "jsonpath", "~> 1.1"
```